tst/EnergyPlus/unit/BaseboardRadiator.unit.cc

```diff
@@ -387,7 +387,7 @@ TEST_F(EnergyPlusFixture, BaseboardConvWater_SizingTest)
 
     DataZoneEquipment::GetZoneEquipmentData1(state);
     // get electric baseboard inputs
-    BaseboardRadiator::GetBaseboardInput(state.dataBaseboardRadiator);
+    BaseboardRadiator::GetBaseboardInput(*state.dataBaseboardRadiator);
```

Comment:
I'm going to clean up these usages and just pass state around so it will be much cleaner.